libraries/shared/storage/backfill/storage_value_loader.go

```diff
@@ -62,10 +61,7 @@ func (r *StorageValueLoader) getStorageKeys() (map[common.Address][]common.Hash,
 	addressToKeys := make(map[common.Address][]common.Hash)
 	for _, i := range r.initializers {
 		transformer := i(r.db)
-		keysLookup, ok := transformer.GetStorageKeysLookup().(storage.KeysLookup)
-		if !ok {
-			return addressToKeys, fmt.Errorf("%v type incompatible. Should be a storage.KeysLookup", keysLookup)
-		}
+		keysLookup := transformer.GetStorageKeysLookup()
```

Comment:
Is it possible for this to return `nil`? I guess that would probably be in a catastrophic config situation, but maybe worth still tossing up an error if that happens?

Reply:
Yep, great idea.

Reply:
Looking at it a bit closer, we'd actually panic if the `StorageKeysLookup` wasn't set on the Transformer before making it here because when creating a new transformer we call `SetDB` on the `StorageKeysLookup` in `NewTransformer`. 🤔 

Not sure I want to make this change now, but I wonder if we should consider returning an error from `NewTransformer` if either `StorageKeysLookup` or `Repository` are nil?

Reply:
Sounds good 👍